tests/conftest.py

```diff
@@ -152,10 +162,14 @@ def app(app):
         draft_action_bp = DraftActionResource(
             service=_draft_service()
         ).as_blueprint("draft_action_resource")
+        draft_version_bp = DraftVersionResource(
+            service=_draft_service()
+        ).as_blueprint("draft_version_resource")
 
         app.register_blueprint(record_bp)
         app.register_blueprint(draft_bp)
         app.register_blueprint(draft_action_bp)
+        app.register_blueprint(draft_version_bp)
```

Comment:
Shelve: We will need to think about a better way to incorporate actions or one-off endpoint in flask-resources... Clearly having to create a new resource each time highlights that we are missing something.

Reply:
We might want to provide yes-another-level of abstraction. Sort of `as_blueprint(actions=True, version=True)` or something like that.